staging/src/k8s.io/client-go/dynamic/simple.go

```diff
@@ -21,9 +21,12 @@ import (
 	"errors"
 	"fmt"
 	"io"
+	"k8s.io/apimachinery/pkg/util/diff"
```

Comment:
Nit: put the golang native packages in one group, and k8s packages as another group.

Reply:
hmm. I wonder why go fmt did not rearrange it.